sources/packages/web/src/views/institution/locations/confirmation-of-enrollment/COESummaryData.vue

```diff
@@ -100,10 +136,38 @@ export default {
     };
 
     const updateSummaryList = async (locationId: number) => {
-      applications.value = await ConfirmationOfEnrollmentService.shared.getCOESummary(
+      const disbursementAndCount = await ConfirmationOfEnrollmentService.shared.getCOESummary(
         locationId,
         props.enrollmentPeriod,
+        {
+          page: page.value,
+          pageLimit: pageLimit.value,
+          sortField: sortField.value,
+          sortOrder: sortOrder.value,
+          searchCriteria: searchCriteria.value,
+        },
       );
+      disbursements.value = disbursementAndCount;
+    };
+
+    const pageEvent = async (event: any) => {
```

Comment:
Is there a reason why we are not defining an interface for the event here?

Reply:
Interface created.

Reply:
Thanks for creating the interface. I just believe that we do not need to append DTO for this object.